src/Cache/Engine/FileEngine.php

```diff
@@ -311,6 +311,8 @@ protected function _clearDirectory($path, $now, $threshold)
 
             try {
                 $file = new SplFileObject($path . $entry, 'r');
+            } catch (\Throwable $e) {
```

Comment:
Oh, ignore. you meant `use Throwable` (like the rest of the global-class use statements) :P